modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/facet/DefaultFacetTranslator.java

```diff
@@ -198,6 +198,11 @@ private QueryBuilder _translateBooleanClause(
 	private final FacetProcessor<SearchRequestBuilder> _defaultFacetProcessor =
 		new FacetProcessor<SearchRequestBuilder>() {
 
+			@Override
+			public String getFacetClassName() {
```

Comment:
@W2jx Please make it clear that this implementation can return null because it's not a service.